client/login/wp-login/style.scss

```diff
@@ -350,17 +350,20 @@ $image-height: 47px;
 		}
 	}
 
+	.login__form-action .components-button.is-primary {
+		margin: 6px auto 13px;
+	}
+
 	.login form {
 
 		// We shouldn't regularly use .components-* classnames. The following is
 		// an exception to opt-out from some style overrides. Ideally, all
 		// input instances would come from `@wordpress/components` and the
 		// following overrides could be safely deleted.
 		input:not(.components-text-control__input),
-		button:not(.social-buttons__button) {
```

Comment:
There are still instances of `social-buttons__button` in the code, right? Is it ok for these styles to now be applied to them?

Reply:
Yes, but they are also a core button now so I'm actually expanding the scope here to include them and the primary buttons